pkg/utils/common.go

```diff
@@ -178,7 +178,17 @@ func SyncData(log sdkTypes.KairosLogger, runner v1.Runner, fs v1.FS, source stri
 	}
 
 	log.Infof("Starting rsync...")
-	args := []string{"--progress", "--partial", "--human-readable", "--archive", "--xattrs", "--acls"}
+	args := []string{
+		"--progress",
+		"--partial",
+		"--human-readable",
+		"--archive",
+		"--acls",
+		"-t",
+		"-U",
+		"-o",
+		"-g",
```

Comment:
Better use the long version of these flags.

Reply:
well, I was gonna put comments on what the flags do in the line after testing that all seems to work, will that work instead of long flags?